link-verifier/verify-links.py

```diff
@@ -14,7 +14,7 @@
 import traceback
 from collections import defaultdict
 
-MARKDOWN_SEARCH_TERM = r'\.md$'
+MARKDOWN_SEARCH_TERM = r"\.md$"
 # Regex to find a URL
 URL_SEARCH_TERM = r'(\b(https?)://[^\s\)\]\\"<>]+[^\s\)\.\]\\"<>])'
```

Comment:
It would make sense to update this regex to exclude the trailing slash, or coma, but I honestly have no idea how this even matches currently.